packages/commons/test/BundledAxeScriptProviderTest.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Deque.AxeCore.Commons;
+using Moq;
+using FluentAssertions;
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Deque.AxeCore.Commons.Test
+{
+    [TestFixture]
+    public class BundledAxeScriptProviderTest
+    {
+        private static readonly string testContext = TestContext.CurrentContext.TestDirectory.ToString();
+        private readonly string basePath = testContext.Split(new string[] { "test" }, StringSplitOptions.None)[0];
+        private static readonly string copyrightHeader = "* Your use of this Source Code Form is subject to the terms of the Mozilla Public\n * License, v. 2.0.";
+        [Test]
+        public void GetScriptCalled()
+        {
+            var scriptProvider = new BundledAxeScriptProvider();
+            Assert.DoesNotThrow(() =>
```

Comment:
In general, it's not necessary to wrap other assertions inside `Assert.DoesNotThrow()` - the test is going to fail anyway if an unexpected exception is thrown, and if the inner assertions fail, the wrapping assertion will make the failure message a little harder to understand. I'd recommend reserving `Assert.DoesNotThrow()` only for cases where there's no reasonable inner assertion you can make.